app/controllers/press-releases/press-release.js

```diff
@@ -71,6 +71,8 @@ export default class PressReleasesPressReleaseController extends Controller {
       publicationEvent.plannedStartDate = publicationDate;
     }
     yield publicationEvent.save();
+    this.model.pressRelease.publicationEvent = publicationEvent;
+    yield this.model.save();
```

Comment:
Dit is niet nodig. Bij de `createRecord` wordt hen publication-event al gelinkt aan het persbericht. Ember Data begrijpt deze inverse. De omgekeerde richting moet daarom niet expliciet gezet en gesaved worden.

Reply:
met de implementatie van de serializer en het weghalen van deze 2 lijnen, word er geen relatie meer opgeslagen bij een persbericht als er een event gemaakt word. Ik doe denk ik alles hoe ik het zou moeten doen, maar het lukt niet

Reply:
Wat bedoel je met "word er geen relatie meer opgeslagen bij een persbericht als er een event gemaakt word"?

Bij het opslaan van het publication-event op lijn https://github.com/kanselarij-vlaanderen/frontend-persberichten/blob/feature/pers-103/app/controllers/press-releases/press-release.js#L73 zou je in de request body van de POST netwerk request om het publication-event aan te maken moeten zien dat het gerelateerde persbericht mee gestuurd wordt. Is dat niet het geval?

Reply:
Oorzaak gevonden. Het publication-event werd nog niet correct aangemaakt en gesaved. Verbeterd in https://github.com/kanselarij-vlaanderen/frontend-persberichten/commit/e23b4ff2c4cabb8a0c6c410cc98df66fa5303617 dus als je master binnenhaalt, werkt dit zonder de extra save aan de kant van persbericht.